sw/airborne/modules/sensors/airspeed_ms45xx_i2c.c

```diff
@@ -56,17 +56,32 @@ PRINT_CONFIG_MSG("USE_AIRSPEED_MS45XX automatically set to TRUE")
 #define MS45XX_I2C_DEV i2c2
 #endif
 
-/** Sensor I2C slave address (defaults 0x50, 0x6C and 0x8C) */
+/** Sensor I2C slave address (existing defaults 0x50, 0x6C and 0x8C)
+ */
 #ifndef MS45XX_I2C_ADDR
 #define MS45XX_I2C_ADDR 0x50
 #endif
 
-/** MS45xx pressure range in psi.
- * The sensor is available in 1, 2, 5, 15, 30, 50, 100, 150 psi ranges.
- * 1 psi max range should be ~100m/s max airspeed.
+/** MS45xx sensors pressure output type can be in PSI or InH2O, as defined in the datasheet
+ *  if not defined to be MS45XX_PRESSURE_OUTPUT_TYPE_InH2O then PSI is used
+ * */
+#ifndef MS45XX_PRESSURE_OUTPUT_TYPE_InH2O
+#define MS45XX_PRESSURE_OUTPUT_TYPE_InH2O 1
+#endif
+
+/** MS45xx pressure range in PSI or InH2O
+ * The sensor is available in many ranges, the datasheet of your pressure sensor will tell which one
+ * and what this range represents
  */
 #ifndef MS45XX_PRESSURE_RANGE
-#define MS45XX_PRESSURE_RANGE 1
+#define MS45XX_PRESSURE_RANGE 4
```

Comment:
I'd rather not change the default range since planes using this rely on it

Reply:
Fine, grep'ed for others using this could not find one, but 1 is fine since Diso sets the default to 4